src/avram/base_query_template.cr

```diff
@@ -40,7 +40,7 @@ class Avram::BaseQueryTemplate
         {% end %}
 
         {% if columns.map(&.[:name].stringify).includes?("updated_at") %}
-          _changes[:updated_at] = Time.adapter.to_db(Time.utc) if updated_at == IGNORE
+          _changes[:updated_at] = Time.adapter.to_db(Time.utc) if updated_at.is_a?(Avram::Nothing)
```

Comment:
oohhh :joy:  I see what it's doing now. That's funny that changing it to this made it clearer for me. This is just saying that if you pass in your own updated_at value, then this won't run because your own updated_at won't be a `Nothing`, it'll be a `Time`. Good call on that :+1: